package.json

```diff
@@ -201,15 +201,13 @@
     "object-path": "^0.11.8",
     "openrosa-formlist": "github:medic/openrosa-formlist#sax",
     "pass-stream": "^1.0.0",
-    "pouchdb-adapter-http": "^7.3.1",
-    "pouchdb-browser": "^7.3.1",
-    "pouchdb-core": "^7.3.1",
-    "pouchdb-debug": "^7.2.1",
-    "pouchdb-find": "^7.3.1",
-    "pouchdb-generate-replication-id": "^7.3.1",
-    "pouchdb-mapreduce": "^7.3.1",
-    "pouchdb-replication": "^7.3.1",
-    "pouchdb-session-authentication": "^1.1.0",
+    "pouchdb-adapter-http": "^9.0.0",
+    "pouchdb-browser": "^9.0.0",
+    "pouchdb-core": "^9.0.0",
+    "pouchdb-find": "^9.0.0",
+    "pouchdb-generate-replication-id": "^9.0.0",
+    "pouchdb-mapreduce": "^9.0.0",
+    "pouchdb-replication": "^9.0.0",
```

Comment:
Can you remove the line in the `dependabot.yml` file that is tagged with #9534?  Should not be anything "blocking" upgrades to Pouch at this point.

Reply:
done